Show files count

blackdaemon opened the pull request:
Add count of displayed files to a "count" section.
Colors the count bright-yellow if a filter is active.

blackdaemon committed:
Add total number of files into 'count' section

blackdaemon committed:
Undo adding symlink to init.lua

imsi32 commented:
I like the idea, it will probably used by someone. However, I think it would be better if it was separate component as there will be others that they don't need it.

There are also things that needs to be changed.For example:

- color (when filter is active, it looks same with the selected count),
- default icon (it is thinner than others and looks similar to copy icon) (it will be better if its one of the nf-fa icons),
- code layout (color and icon definition needs to be like other definition and should provide config)

For reference:
![resim](https://github.com/user-attachments/assets/b09e54bb-b7cf-4e63-a382-b2770b56bbd5)
![resim](https://github.com/user-attachments/assets/2c5a42ab-7f6c-44ee-8b7a-5eadc53f5b75)

If you want to make this changes, I can wait for it. Or I can do the clean up.

blackdaemon commented:
Hi, 
- color: Yes, didn't try that in non-truecolor terminal, I will change that (see below how it looks on mine with brightyellow)
- icon: I did not find any 'list of files' related icon in nf-fa other than already used, but `nf-fa-buffer` looks quite fitting, what do you think?
![image](https://github.com/user-attachments/assets/98f145c0-96d4-4f73-978d-fb5d18b8a932)
- I will update the code layout and config

Regarding making it separate component, yes, I can add it as separate, however as it is still a count of something, would fit nicely into count coloreds. What about making it switchable on/off in the config?

blackdaemon commented:
As of how the icons sometimes looking similar to each other, it really depends on the nerdfonts font used, each of them have slightly different approach, the icon I used originally looks completely different in my font than what you have in your screenshot. 
Do you have particular font in mind as a reference?

imsi32 commented:
I think, it can be set on/off using parameter in count. However, can it be only name like `filter'. (not `config` and accessed with `config.filter`).

Also, I like the icon idea. We can use two icons: `nf-fa-buffer` and `nf-fa-filter` looks good.

The colors had a problem on my system but I realized that these colors are used in `section a` so it will be better if it uses `blue` for normal and `magenta` for filtered.

imsi32 committed:
[FEAT] Number of files or filtered files

details: count component can now show number of files or filtered files
if it gets filter parameter as true
